pv211_utils/beir/leaderboard.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+import pkg_resources
+from typing import Optional
+
+from ..leaderboard import GoogleSpreadsheetLeaderboardBase
+
+
+class BeirLeaderboard(GoogleSpreadsheetLeaderboardBase):
+    def get_competition_start(self) -> datetime.date:
+        return datetime.date(2022, 8, 18)
+
+    def get_competition_end(self) -> datetime.date:
+        return datetime.date(2022, 12, 30)
```

Comment:
Should be updated to reflect the submission dates for the second term project in [the interactive syllabus][1]. Same for `arqmath/leaderboard.py`.

 [1]: https://is.muni.cz/auth/el/fi/jaro2023/PV211/index.qwarp

```suggestion
    def get_competition_start(self) -> datetime.date:
        return datetime.date(2023, 3, 22)

    def get_competition_end(self) -> datetime.date:
        return datetime.date(2023, 4, 30)
```

Reply:
should be done